src/main/java/org/prnhs/javaee/swim/core/dto/ContactsDto.java

```diff
@@ -0,0 +1,93 @@
+package org.prnhs.javaee.swim.core.dto;
+
+import java.util.Objects;
+
+public class ContactsDto {
+    
+    Integer id;

```

Comment:
always specify private, otherwise the default is protected (which is package protected... other kids in the same package can see your toys)